backend/src/main/java/com/elshop/backend/category/CategoryController.java

```diff
@@ -22,6 +21,24 @@ public Category addNewCategory(@Valid @RequestBody CategoryRequest requestData)
         return categoryService.add(requestData);
     }
 
-    ;
+    @GetMapping
+    public List<Category> addNewCategory() {
+        return categoryService.getAll();
+    }
+
+    @GetMapping("{id}")
+    public Category addNewCategory(@PathVariable String id) {
```

Comment:
I think the name of the method is wrong?

Reply:
right, I will fix it